docs/includes/generated_docs/language__series.md

```diff
@@ -3219,15 +3219,14 @@ school_year = patients.age_on("2020-09-01").map_values(
   <a class="headerlink" href="#MultiCodeStringPatientSeries.contains" title="Permanent link">🔗</a>
 </div>
 <div markdown="block" class="indent">
-Check if the multi code field contains a specific code string and
-return the result as a boolean series. `code` can
-either be a string (and prefix matching works so e.g. "N17" in ICD-10
-would match all acute renal failure), or a clinical code.
+Check if the multi code field contains a specific code string and return the
+result as a boolean series. `code` can a partial match (so e.g. "N17" in ICD-10
```

Comment:
Typo: should `can a partial match` be `can be a partial match`?

Ditto for a few instances below.

Reply:
Ah, thanks! The actual typo here is in the docstring from which these docs are generated, which is why it's repeated multiple times.

All the files in `docs/includes/generated_docs` are autogenerated from the code. There's a command `just generate-docs` which updates these. And CI enforces that they're always up-to-date.

We generally try to keep all the autogenerated docs changes together in a single commit to make life easier for reviewers.